pkg/kubelet/cm/dra/plugin/plugin.go

```diff
@@ -150,7 +199,7 @@ func (h *RegistrationHandler) validateVersions(
 }
 
 func unregisterPlugin(pluginName string) {
```

Comment:
can we call this `deregisterPlugin` to match the top-level function that calls it?

Reply:
done in b55b0297bad